app/models/claims/mentor_training.rb

```diff
@@ -31,6 +31,11 @@ class Claims::MentorTraining < ApplicationRecord
 
   audited associated_with: :claim
 
+  enum :training_type, {
+    initial: "initial",
+    refresher: "refresher",
+  }, default: :initial, validate: { allow_nil: false }
```

Comment:
```suggestion
  }, default: :initial, validate: true
```

This will validate that any value is one of the enum values, `nil` will be invalid. Ref: https://blog.saeloun.com/2024/01/02/rails-enums-validate-option/#after